egs/librispeech_french/asr1/run.sh

```diff
@@ -0,0 +1,306 @@
+#!/bin/bash
+
+# Copyright 2018 Kyoto University (Hirofumi Inaguma)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch # chainer or pytorch
+stage=-1        # start from -1 if you need to start from data download
+stop_stage=100
+ngpu=1          # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump    # directory to dump full features
+N=0             # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0       # verbose option
+resume=         # Resume the training from snapshot
+seed=1          # seed to generate random number
+# feature configuration
+do_delta=false
+
+train_config=conf/tuning_asr/train_rnn_char.yaml
+lm_config=conf/tuning_asr/lm_char.yaml
+decode_config=conf/tuning_asr/decode_rnn_char.yaml
+
+# rnnlm related
+lm_resume=        # specify a snapshot file to resume LM training
+lmtag=            # tag for managing LMs
+
+# decoding parameter
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+
+# preprocessing related
+case=lc.rm
+# tc: truecase
+# lc: lowercase
+# lc.rm: lowercase with punctuation removal
+
+# Set this to somewhere where you want to put your data, or where
+# someone else has already put it.  You'll want to change this
+# if you're not on the CLSP grid.
```

Comment:
This path is not CLSP one.

Reply:
@sw005320  Could you tell me the absolute path where you downloaded data?

Reply:
We don't finish the download yet, and I can do it later. So, you can just replace the above `CLSP grid` with your lab's grid (or remove this sentence).